testdata/kata/release-4.10/subscription.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
All the content of subscription.yaml already part of deployment.yaml, are they both needed/used ?

Reply:
The subscription.yaml is not used in this scenario, but will be used by other scenarios with step "Given the kata-operator is installed using OLM GUI"
In thie scenario, we used deployment.yaml which is required by step "Given the kata-operator is installed using OLM CLI"